experimenter/experimenter/jetstream/client.py

```diff
@@ -244,6 +244,7 @@ def get_experiment_data(experiment: NimbusExperiment):
                 experiment_data[window][AnalysisBasis.ENROLLMENTS] = {}
                 raw_data[window][AnalysisBasis.ENROLLMENTS] = {}
             elif point["analysis_basis"] == AnalysisBasis.EXPOSURES:
+                experiment_data["has_exposures"] = point.get("point", 0) > 0
```

Comment:
I don't think this solves the issue I mentioned previously. It will still set this value based on the last data point in the list, and it will be the same for all windows.